### DISCUSSION TITLE
How can I access public bucket files after migration with dump file

### DISCUSSION THREAD

**Discussion by DevYuns** [supabase/supabase#5285] [Questions] [answered]
Hey. I'm new to supabase. I got a problem to access files in public bucket.

To migrate data from old supabase project to new supabase project easily, I tried to backup whole database directly with `dbeaver`  in my local environment, then restore it to the new project database. All data was restored include bucket files without problem. But when I try to access the files with `public url`, I got a message `access denied`. The buckets are public and there are no other restrict. What should I do to get a right public url within new a supabase project?

Thank you in advance.

**Comment by GaryAustin1** [comment, marked as answer]
At some point Supabase added another column to storage, maybe your old project was before this change...]

https://github.com/supabase/storage-api/blob/master/migrations/tenant/0007-add-public-to-buckets.sql
ALTER TABLE storage.buckets ADD COLUMN "public" boolean default false;

**Reply by DevYuns** [reply]
@GaryAustin1 Thank you for quick reply! I checked the `storage.buckets` and I found that `public` set to `false`.
Is there any way to switch it into `true` with supabase?

**Reply by GaryAustin1** [reply]
With the UI table editor or sql window.

**Reply by GaryAustin1** [reply]
I am curious though you say bucket files...  I seriously doubt any s3 files were actually copied.... just the storage database info which is not that useful without the file....

**Reply by DevYuns** [reply]
oops I got a mistake. `buckets` are already set to `true`.

I agree with your doubt. I'm not sure what I did 😓 .

**Reply by GaryAustin1** [reply]
Not anything you did.  The actual file data is not in the database.  All you can copy over is your bucket structure.  Anything else has to be done by hand or script to copy files off of old system to new.

**Reply by DevYuns** [reply]
I already deleted the old project. Can I restore it?

**Reply by GaryAustin1** [reply]
Unfortunately no.  I've seen the suggestion to ask support, but never seen the outcome of that in a response later.  Probably depends on length of time if at all.

**Reply by DevYuns** [reply]
Ok. Thank you for your help. I learned a great lesson 😭 .
